graylog2-web-interface/src/components/maps/configurations/GeoIpResolverConfig.tsx

```diff
@@ -195,6 +199,14 @@ const GeoIpResolverConfig = ({ config = defaultConfig, updateConfig }: Props) =>
                                  hideCheckbox
                                  units={['SECONDS', 'MINUTES', 'HOURS', 'DAYS']} />
 
+                  <Row>
+                    <Col sm={6}>
+                      <FormikInput id="use_s3"
+                                   type="checkbox"
```

Comment:
Nit but the S3 checkbox stays enabled when "enable processor" is unchecked.
```suggestion
                                   type="checkbox"
                                   disabled={!values.enabled}
```

Reply:
Good catch. The `Enforce Graylog schema` box acts the same so I am going to add this there as well.